sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/session.hpp

```diff
@@ -208,12 +208,20 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
     /** @brief Begins operations on the session.
      *
      */
-    void Begin();
+    void Begin(Azure::Core::Context const& context = {});
```

Comment:
Given this is all internal and not customer facing, do you want to drop the default (`= {}`) value for the context, to make sure an SDK dev forgets to pass a context here?
(Should you go with this suggestion, there is more than this place that uses the default value)

Reply:
That's an interesting observation, and yeah, I think it makes sense because the callers should always have a Context, and this functions as a forcing function if they don't.